collect_app/src/main/res/values/strings.xml

```diff
@@ -611,4 +613,24 @@
     <string name="instance_upload_cancelled">Instance upload canceled</string>
     <string name="hide_old_form_versions_setting_title">Hide old form versions</string>
     <string name="hide_old_form_versions_setting_summary">Only the newest version will appear in Fill Blank Form</string>
+    <string name="sms_pref_dialog_title">Destination Phone Number</string>
+    <string name="sms_pref_title">Phone Number</string>
+    <string name="sms_invalid_phone_number">Invalid phone number!</string>
+
+    <!-- Sms submission -->
+    <string name="sms_no_reception">\'Failed on\' EEE, MMM dd, yyyy \'at\' HH:mm \'due to no reception\'</string>
```

Comment:
Oh never thought of this! Just didn't really think users would be switching languages while using the app. Yes, so the right solution would be for me to pull the status and date and then generate the status from that instead of persisting it. Gonna look a way to do this.

Reply:
I don't think it affects behavior currently, right? It just means that status messages could be in different languages which is weird but not really all that bad. I think it makes sense to leave as-is for this PR and come back with a more complete solution in #1419.

Reply:
No it doesn't affect the current behaviour. Only if the user switches the language.

Reply:
The question is whether there's ever any attempt to read status from the status string. If so, that's a problem. But I don't believe it is the case.